backend/dal/Migrations/20210128205552_v01.06.05.cs

```diff
@@ -0,0 +1,22 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+using Pims.Dal.Helpers.Migrations;
+
+namespace Pims.Dal.Migrations
+{
+    public partial class v010605 : SeedMigration
```

Comment:
Make sure you merge after `v01.06.04` is merged.  It may already be, I haven't checked.